source/extensions/omni.isaac.groundcontrol/omni/isaac/groundcontrol/devices/keyboard/se2_keyboard.py

```diff
@@ -153,11 +153,11 @@ def _create_key_bindings(self):
             "NUMPAD_2": np.asarray([-1.0, 0.0, 0.0]) * self.v_x_sensitivity,
             "DOWN": np.asarray([-1.0, 0.0, 0.0]) * self.v_x_sensitivity,
             # right command
-            "NUMPAD_4": np.asarray([0.0, 1.0, 0.0]) * self.v_y_sensitivity,
-            "LEFT": np.asarray([0.0, 1.0, 0.0]) * self.v_y_sensitivity,
+            "NUMPAD_4": np.asarray([0.0, -1.0, 0.0]) * self.v_y_sensitivity,
+            "LEFT": np.asarray([0.0, -1.0, 0.0]) * self.v_y_sensitivity,
             # left command
-            "NUMPAD_6": np.asarray([0.0, -1.0, 0.0]) * self.v_y_sensitivity,
-            "RIGHT": np.asarray([0.0, -1.0, 0.0]) * self.v_y_sensitivity,
+            "NUMPAD_6": np.asarray([0.0, 1.0, 0.0]) * self.v_y_sensitivity,
+            "RIGHT": np.asarray([0.0, 1.0, 0.0]) * self.v_y_sensitivity,
```

Comment:
Thanks! I should check that this is correct in the other devices as well, but merging this now!